keps/prod-readiness/sig-release/2572.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 2572
```

Comment:
is this the right number?

Reply:
@dims this one is the PRR approver. I *think* the PRR is not actually relevant, at least not in the current form, for this KEP but the existing validation required it.

Reply:
Indeed, the KEP number corresponds to the issue number. I included the wrong one in my email 😬  

The tracking issue is here: https://github.com/kubernetes/enhancements/issues/2572